gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/EhrResourceExtractorUtil.java

```diff
@@ -63,6 +78,17 @@ private static boolean validObservationStatement(RCMRMT030101UK04Component4 comp
         return component.getObservationStatement() != null && component.getObservationStatement().getId() == resourceId;
     }
 
+    private static boolean filterForValidObservationStatements(RCMRMT030101UK04EhrComposition ehrComposition) {
```

Comment:
if this is a method being kept in the EhrExtractResourceUtil class, should this method not be named more appropriately for filter for valid Immunization Observation Statements? e.g. `filterForValidImmunizationObservationStatements`?

Reply:
Good point, done